_docs/sysadmin/system_customization.md

```diff
@@ -215,11 +247,48 @@ memory_limit
 Just be aware that modifying this number can have repercussions when multiple
 students are using the system at once.
 
-### Show system message to all users
+<small>[Back To Table of Contents](#toc)</small>
+
+
+### <a name="system_messages">Show system message to all users</a>
 
 Submitty allows showing a message to all users on all pages. This is useful for advertising
 events that affect all users, such as system maintainence windows where it would be unavailable.
 This message is shown in a yellow bar displayed underneath the header.
 
 To add or remove this message, edit `/usr/local/submitty/config/submitty.json` and add/remove the
 key/value for `system_message`. If the key exists, but is empty, no message will be shown.
+
+<small>[Back To Table of Contents](#toc)</small>
+
+
+### <a name="footer_links">Adding Additional Links To The Footer</a>
+
+You may add additional links to be shown in the footer.  i.e. you may link to pages related to your institution or public policy notices.
+Additional links will appear to the right of the copyright notice and credit links to Github and RCOS.
+
+1. Create an empty file `footer_links.json` and place it in Submitty's system configuration folder.
+  * The configuration folder is typically `/usr/local/submitty/config`.
+2. The `footer_links.json` file must be readable by the `submitty_php` user.
+  * e.g. Set ownership of the file to `root:submitty_php` and permissions to `RW-R-----`.
+3. Every link requires a `title` property and a `url` property.
+  * An `icon` property is optional.  Check [here](https://fontawesome.com/v4.7.0/icons/) for a list of icons.
+4. Here is an example `footer_links.json`.  Make sure your JSON file is properly formatted with square brackets, curly braces, commas, etc.
+```
+[
+    {
+        "title": "Dept of Computer Science",
+        "url": "https://www.myuniversity.edu/computer_science"
+    },
+    {
+        "icon": "universal-access",
+        "title": "Accessibility Policy",
+        "url": "https://www.myuniversity.edu/info/accessibilitypolicy"
+    }
+]
+```
+5. If any links do not display, they probably have failed validation.  Validation can be particular, so please carefully proofread `footer_links.json` with instructions 1—4.
```

Comment:
I think we can maybe get rid of this step, as it's mostly just (to me) "re-read instructions 1-4 if things fail". We can probably append to step 3 that "any object missing a title or url will not be shown" or something like that?